components/alert/alert.css

```diff
@@ -0,0 +1,74 @@
+
+@import "/reset.css";
+.alert-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /* max-width: 100%; */
+    /* height: auto; */
+    align-items: center;
+    gap: 30%;
+}
+
+.alert-box{
+    border-radius: 0.3rem;
+    padding: 1rem 3rem 1rem 1rem;
+    font-size: 1rem;
+    box-sizing: border-box;
+    width: 100%;
+    margin: 1rem 0;
+    position: relative;
+}
+.alert-primary{
+    background-color: var(--alert-primary);
+    color: #1c316b;
+}
+.alert-success{
+    background-color: var(--success-color);
+    color: #0a4d35;
+}
+.alert-danger{
+    color:  #ee2626;
+    background-color: var(--danger-color);
+}
+.alert-warning{
+    color: #b16501;
+    background-color: var(--warning-color);
+}
+/* 
+CSS FOR OULINE ALERT
+*/  
+.alert-primary-outline{
+    background-color: #ffffff;
+    color: #1c316b;
+    border: 2px solid blue;
+}
+.alert-danger-outline{
+    color: #ee2626;
+    background-color: #ffffff;
```

Comment:
Background-color property is repeated. Try to use a descendant selectors or use common class name.